Gemfile.lock

```diff
@@ -5,31 +5,31 @@ GEM
       public_suffix (>= 2.0.2, < 5.0)
     colorator (1.1.0)
     concurrent-ruby (1.1.7)
-    em-websocket (0.5.1)
+    em-websocket (0.5.2)
```

Comment:
Same as the `Gemfile`, you do not need to push the edited file to your project repo. It is fine for your local machine when testing and developing.